### DISCUSSION TITLE
Branch/PR checks

### DISCUSSION THREAD

**Discussion by HyperBrain** [serverless-heaven/serverless-webpack#1540] [Team Posts]
I found out that due to GitHub changes the previously enabled checks were disabled. I re-enabled everything. This inevitably leads to new checks of the currently open PRs which have to be fixes first.
Now Travis (including ESLint runs), AppVeyor and Coveralls have to pass again before anything can be merged. This re-establishes the quality gates again.

**Comment by hassankhan** [comment]
Not quite related, but do you think its worth using a [GitHub project](https://help.github.com/en/articles/about-project-boards)? The "Automated kanban with review" board in particular seems useful, though I haven't used it personally.

**Comment by HyperBrain** [comment]
I used it with serverless-aws-alias. However, the boards are most effective for feature implementations. Having the issues with correct assignments can be sufficient for now - if we encounter incoming bursts of bugs/features, we can switch to a board with prioritization later. What do you think?
